post_processing/orac_postproc.F90

```diff
@@ -492,54 +520,43 @@ subroutine orac_postproc(mytask, ntasks, lower_bound, upper_bound, &
                verbose)
        end if
 
-       ! Read fields that vary from file to file
-       if (.not. use_new_bayesian_selection) then
-          do i = 1, n_in_files
-             if (verbose) write(*,*) '********************************'
-             if (verbose) write(*,*) 'read: ', trim(in_files_primary(i))
-             call alloc_input_data_primary_class(loop_ind(i), input_primary(i))
-             call read_input_primary_class(in_files_primary(i), &
-                  input_primary(i), loop_ind(i), .False., &
-                  chunk_starts(i_chunk), verbose)
-             if (do_secondary) then
-                if (verbose) write(*,*) 'read: ', trim(in_files_secondary(i))
-                call alloc_input_data_secondary_class(loop_ind(i), &
-                     input_secondary(i))
-                call read_input_secondary_class(in_files_secondary(i), &
-                     input_secondary(i), loop_ind(i), chunk_starts(i_chunk), &
-                     verbose)
-             end if
-          end do
-       else ! Use traditional selection method
-          ! Load only the cost values from input files
-          do i = 1, n_in_files
-             call alloc_input_data_only_cost(loop_ind(i), input_primary(i), &
```

Comment:
I think the `_only_cost` routines are no longer called after this commit. In the name of tidiness, I'd be inclined to remove them, and get rid of the corresponding argument from `read_input_primary_class()`

Reply:
Agreed.